common/reviews/api/gulp-core-build-webpack.api.ts

```diff
@@ -7,19 +7,16 @@ interface IWebpackTaskConfig {
 }
 
 // @public (undocumented)
-class WebpackTask extends GulpTask<IWebpackTaskConfig> {
+class WebpackTask<TExtendedConfig = {}> extends GulpTask<IWebpackTaskConfig & TExtendedConfig> {
+  constructor(extendedName?: string, extendedConfig?: TExtendedConfig);
   // (undocumented)
   public executeTask(gulp: typeof Gulp, completeCallback: (error?: string) => void): void;
   // (undocumented)
   public isEnabled(buildConfig: IBuildConfig): boolean;
   // (undocumented)
   public loadSchema(): Object;
   // (undocumented)
-  public name: string;
```

Comment:
we also manually set this at times... why remove these properties? they should still be publicly accessible